### PR TITLE
LPS-173420 portal-search-web: Add character threshold to basic + apply helptext

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-173420 - [FE] Update help text within suggestion contributor

Added the helptext for the character threshold and updated helptext for suggestion contributor! The character threshold has also been added to basic options, as a second line (see third image). 

Edit: Added an update to `_getLowestSuggestionsDisplayThreshold` so that it decides thresholds for each contributor, and then it takes the minimum of that. I think the previous way was disregarding the global one if any individual threshold was set.

Screen Shot 2023-01-23 at 4 07 02 PM

Screen Shot 2023-01-23 at 3 46 16 PM

Screen Shot 2023-01-23 at 4 05 48 PM

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:search

**Comment by oliv-yu** [comment]
ci:test:sf

**Comment by oliv-yu** [comment]
Hm, not sure about the recent number of test failures but QA had mentioned they were taking a look at it. I'll assign this for review for the time being, hopefully the tests results go back to normal soon. Please let me know what you think, thanks!

**Review by thektan — CHANGES_REQUESTED** [review]
Changes look good! I did notice this doc needed to be updated though. Can you update this to reflect the parameter being passed in? https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/utils/clean_suggestions_contributor_configuration.js#L21

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/utils/clean_suggestions_contributor_configuration.js [hunk 1]
```diff
@@ -16,9 +16,10 @@ import parseJSONString from './parse_json_string';
 import {CONTRIBUTOR_TYPES} from './types/contributorTypes';
 
 /**
- * Filters out blueprint suggestion contributors in the fields array
- * if search experiences are not supported.
- * @param {Array} fields The list of fields.
+ * Filters out blueprint suggestion contributors in the
+ * `suggestionsContributorConfiguration` array if search experiences
+ * are not supported.
+ * @param {Array} suggestionsContributorConfiguration
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/utils/clean_suggestions_contributor_configuration.js:22]: `suggestionsContributorConfiguration` is a string. Can you also add a description that this is a string of an array of objects?

Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/utils/clean_suggestions_contributor_configuration.js:22]: Oh my bad, didn't see that, thanks! I'll add it in.

**Comment by thektan** [comment]
ci:forward

**Review by thektan — APPROVED** [review]

**Pull request closed without merging**